localstack-core/localstack/services/lambda_/provider.py

```diff
@@ -1037,6 +1042,13 @@ def create_function(
             )
             fn.versions["$LATEST"] = version
             state.functions[function_name] = fn
+        function_counter.labels(
+            operation=FunctionOperation.create,
+            runtime=runtime,
```

Comment:
@dfangl : Detect `None` and add "n/a" for `package_type=Image`

Suggestion: add `or` "n/a"